sdcm/sct_config.py

```diff
@@ -1091,6 +1106,12 @@ class SCTConfiguration(dict):
         dict(name="scylla_mgmt_upgrade_to_repo", env="SCT_SCYLLA_MGMT_UPGRADE_TO_REPO", type=str,
              help="Url to the repo of scylla manager version to upgrade to for management tests"),
 
+        dict(name="mgmt_restore_params", env="SCT_MGMT_RESTORE_PARAMS", type=dict_or_str_or_pydantic,
+             help="Manager restore operation specific parameters: batch_size, parallel"),
```

Comment:
please update help to include info how to use dict here (showing batch_size and parallel require int values).

Reply:
Fixed